homeassistant/components/srp_energy/const.py

```diff
@@ -1,12 +1,16 @@
 """Constants for the SRP Energy integration."""
 from datetime import timedelta
+import logging
 
-SRP_ENERGY_DOMAIN = "srp_energy"
+LOGGER = logging.getLogger(__package__)
+
+DOMAIN = "srp_energy"
 DEFAULT_NAME = "SRP Energy"
 
 CONF_IS_TOU = "is_tou"
 
 ATTRIBUTION = "Powered by SRP Energy"
+PHOENIX_TIME_ZONE = "America/Phoenix"
```

Comment:
This isn't used.

Reply:
It's used in the tests. HA threw a warning to have consts accessible via the component. See tests/components/srp_energy/conftests.py line 26

Reply:
If it's only used in tests, move it to the tests.